src/OpenTelemetry/CHANGELOG.md

```diff
@@ -17,10 +17,21 @@
   ([#5004](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5004))
   ([#5016](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5016))
 
-* Update `AggregatorStore` to provide known connection metrics with larger
-  histogram buckets.
+* Update Metrics SDK to override the default histogram buckets for the following
+  metrics from ASP.NET Core and HttpClient runtime:
+  * `signalr.server.connection.duration`
+  * `kestrel.connection.duration`
+  * `http.client.connection.duration`
+
+  These histogram metrics which have their `Unit` as `s` (second) will have
+  their default histogram buckets as `[ 0.01, 0.02, 0.05, 0.1, 0.2, 0.5, 1, 2,
+  5, 10, 30, 60, 120, 300 ]`.
   ([#5008](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5008))
 
+* Remove the bucket with value `0` for histogram buckets for metrics from
```

Comment:
do we really need this entry as the above entry already shows the buckets without 0?

Reply:
The PR associated with the above entry adds the zero bucket ☹️

This PR removes the zero bucket for the previously added metrics as well. I thought it's better to just list out the correct buckets even for the above entry.